Enable jumbo build for the following component(s)/source_set(s):

- //net:net

diff -uNr a/net/BUILD.gn b/net/BUILD.gn
--- a/net/BUILD.gn
+++ b/net/BUILD.gn
@@ -8,6 +8,7 @@
 import("//build/config/compiler/compiler.gni")
 import("//build/config/cronet/config.gni")
 import("//build/config/features.gni")
+import("//build/config/jumbo.gni")
 import("//crypto/features.gni")
 import("//net/features.gni")
 import("//testing/libfuzzer/fuzzer_test.gni")
@@ -137,7 +138,7 @@
   flags = [ "CRONET_BUILD=$is_cronet_build" ]
 }
 
-component("net") {
+jumbo_component("net") {
   sources = [
     "base/address_family.cc",
     "base/address_family.h",
@@ -1009,6 +1010,9 @@
     "url_request/websocket_handshake_userdata_key.cc",
     "url_request/websocket_handshake_userdata_key.h",
   ]
+  jumbo_excluded_sources = [
+    "spdy/spdy_stream.cc", # Fail to compile due to function name confliction
+  ]
 
   if (is_posix || is_fuchsia) {
     sources += [ "base/net_errors_posix.cc" ]
@@ -1163,6 +1167,10 @@
       "base/network_interfaces_linux.h",
       "base/platform_mime_util_linux.cc",
     ]
+    jumbo_excluded_sources += [
+      "base/address_tracker_linux.cc",     # Fail to compile due to the confliction between `linux/if.h` and `net/if.h`
+      "base/network_interfaces_linux.cc",  # Ditto
+    ]
   }
 
   if (is_linux) {
diff -uNr a/net/base/host_port_pair.cc b/net/base/host_port_pair.cc
--- a/net/base/host_port_pair.cc
+++ b/net/base/host_port_pair.cc
@@ -20,6 +20,8 @@
 #include "url/gurl.h"
 #include "url/scheme_host_port.h"
 
+#define kValuePortKey kValuePortKey_HostPortPair
+
 namespace net {
 
 namespace {
@@ -139,3 +141,5 @@
 }
 
 }  // namespace net
+
+#undef kValuePortKey
diff -uNr a/net/base/ip_endpoint.cc b/net/base/ip_endpoint.cc
--- a/net/base/ip_endpoint.cc
+++ b/net/base/ip_endpoint.cc
@@ -29,6 +29,8 @@
 #include "net/base/winsock_util.h"  // For kBluetoothAddressSize
 #endif
 
+#define kValuePortKey kValuePortKey_IPEndPoint
+
 namespace net {
 
 namespace {
@@ -234,3 +236,5 @@
 }
 
 }  // namespace net
+
+#undef kValuePortKey
diff -uNr a/net/filter/filter_source_stream.cc b/net/filter/filter_source_stream.cc
--- a/net/filter/filter_source_stream.cc
+++ b/net/filter/filter_source_stream.cc
@@ -17,6 +17,8 @@
 #include "net/base/io_buffer.h"
 #include "net/base/net_errors.h"
 
+#define kDeflate kDeflate_FilterSourceStream
+
 namespace net {
 
 namespace {
@@ -218,3 +220,5 @@
 }
 
 }  // namespace net
+
+#undef kDeflate
diff -uNr a/net/filter/gzip_source_stream.cc b/net/filter/gzip_source_stream.cc
--- a/net/filter/gzip_source_stream.cc
+++ b/net/filter/gzip_source_stream.cc
@@ -17,6 +17,8 @@
 #include "net/base/io_buffer.h"
 #include "third_party/zlib/zlib.h"
 
+#define kDeflate kDeflate_GzipSourceStream
+
 namespace net {
 
 namespace {
@@ -264,3 +266,5 @@
 }
 
 }  // namespace net
+
+#undef kDeflate
diff -uNr a/net/http/http_server_properties_manager.cc b/net/http/http_server_properties_manager.cc
--- a/net/http/http_server_properties_manager.cc
+++ b/net/http/http_server_properties_manager.cc
@@ -27,6 +27,8 @@
 #include "url/gurl.h"
 #include "url/scheme_host_port.h"
 
+#define kVersionKey kVersionKey_HTTPServerPropertiesManager
+
 namespace net {
 
 namespace {
@@ -958,3 +960,5 @@
 }
 
 }  // namespace net
+
+#undef kVersionKey
diff -uNr a/net/http/transport_security_persister.cc b/net/http/transport_security_persister.cc
--- a/net/http/transport_security_persister.cc
+++ b/net/http/transport_security_persister.cc
@@ -28,6 +28,9 @@
 #include "net/http/transport_security_state.h"
 #include "third_party/abseil-cpp/absl/types/optional.h"
 
+#define kVersionKey kVersionKey_TransportSecurityPersister
+#define LoadState   LoadState_TransportSecurityPersister
+
 namespace net {
 
 namespace {
@@ -298,3 +301,6 @@
 }
 
 }  // namespace net
+
+#undef kVersionKey
+#undef LoadState
diff -uNr a/net/network_error_logging/network_error_logging_service.cc b/net/network_error_logging/network_error_logging_service.cc
--- a/net/network_error_logging/network_error_logging_service.cc
+++ b/net/network_error_logging/network_error_logging_service.cc
@@ -32,6 +32,11 @@
 #include "url/gurl.h"
 #include "url/origin.h"
 
+#define kIncludeSubdomainsKey kIncludeSubdomainsKey_NetworkErrorLoggingService
+#define kMaxAgeKey            kMaxAgeKey_NetworkErrorLoggingService
+#define kMaxJsonSize          kMaxJsonSize_NetworkErrorLoggingService
+#define kMaxJsonDepth         kMaxJsonDepth_NetworkErrorLoggingService
+
 namespace net {
 
 namespace {
@@ -1046,3 +1051,8 @@
     : clock_(base::DefaultClock::GetInstance()) {}
 
 }  // namespace net
+
+#undef kIncludeSubdomainsKey
+#undef kMaxAgeKey
+#undef kMaxJsonSize
+#undef kMaxJsonDepth
diff -uNr a/net/quic/bidirectional_stream_quic_impl.cc b/net/quic/bidirectional_stream_quic_impl.cc
--- a/net/quic/bidirectional_stream_quic_impl.cc
+++ b/net/quic/bidirectional_stream_quic_impl.cc
@@ -20,6 +20,8 @@
 #include "net/third_party/quiche/src/quiche/spdy/core/http2_header_block.h"
 #include "quic_http_stream.h"
 
+#define ScopedBoolSaver ScopedBoolSaver_BidirectionalStreamQuicImpl
+
 namespace net {
 namespace {
 // Sets a boolean to a value, and restores it to the previous value once
@@ -409,3 +411,5 @@
 }
 
 }  // namespace net
+
+#undef ScopedBoolSaver
diff -uNr a/net/quic/dedicated_web_transport_http3_client.cc b/net/quic/dedicated_web_transport_http3_client.cc
--- a/net/quic/dedicated_web_transport_http3_client.cc
+++ b/net/quic/dedicated_web_transport_http3_client.cc
@@ -29,6 +29,8 @@
 #include "net/url_request/url_request_context.h"
 #include "url/scheme_host_port.h"
 
+#define HostsFromOrigins HostsFromOrigins_DedicatedWebTransportHTTP3Client
+
 namespace net {
 
 namespace {
@@ -977,3 +979,5 @@
 }
 
 }  // namespace net
+
+#undef HostsFromOrigins
diff -uNr a/net/quic/quic_chromium_client_stream.cc b/net/quic/quic_chromium_client_stream.cc
--- a/net/quic/quic_chromium_client_stream.cc
+++ b/net/quic/quic_chromium_client_stream.cc
@@ -25,6 +25,8 @@
 #include "net/third_party/quiche/src/quiche/quic/core/quic_utils.h"
 #include "net/third_party/quiche/src/quiche/quic/core/quic_write_blocked_list.h"
 
+#define ScopedBoolSaver ScopedBoolSaver_QuicChromiumClientStream
+
 namespace net {
 namespace {
 // Sets a boolean to a value, and restores it to the previous value once
@@ -801,3 +803,5 @@
 }
 
 }  // namespace net
+
+#undef ScopedBoolSaver
diff -uNr a/net/quic/quic_session_pool.cc b/net/quic/quic_session_pool.cc
--- a/net/quic/quic_session_pool.cc
+++ b/net/quic/quic_session_pool.cc
@@ -73,6 +73,8 @@
 #include "url/scheme_host_port.h"
 #include "url/url_constants.h"
 
+#define HostsFromOrigins HostsFromOrigins_QuicSessionPool
+
 namespace net {
 
 namespace {
@@ -2481,3 +2483,5 @@
 }
 
 }  // namespace net
+
+#undef HostsFromOrigins
diff -uNr a/net/reporting/reporting_header_parser.cc b/net/reporting/reporting_header_parser.cc
--- a/net/reporting/reporting_header_parser.cc
+++ b/net/reporting/reporting_header_parser.cc
@@ -25,6 +25,9 @@
 #include "net/reporting/reporting_delegate.h"
 #include "net/reporting/reporting_endpoint.h"
 
+#define kIncludeSubdomainsKey kIncludeSubdomainsKey_ReportingHeaderParser
+#define kMaxAgeKey            kMaxAgeKey_ReportingHeaderParser
+
 namespace net {
 
 namespace {
@@ -351,3 +354,6 @@
 }
 
 }  // namespace net
+
+#undef kIncludeSubdomainsKey
+#undef kMaxAgeKey
diff -uNr a/net/reporting/reporting_service.cc b/net/reporting/reporting_service.cc
--- a/net/reporting/reporting_service.cc
+++ b/net/reporting/reporting_service.cc
@@ -29,6 +29,9 @@
 #include "url/gurl.h"
 #include "url/origin.h"
 
+#define kMaxJsonSize  kMaxJsonSize_ReportingService
+#define kMaxJsonDepth kMaxJsonDepth_ReportingService
+
 namespace net {
 
 namespace {
@@ -347,3 +350,6 @@
 }
 
 }  // namespace net
+
+#undef kMaxJsonSize
+#undef kMaxJsonDepth
diff -uNr a/net/websockets/websocket_http2_handshake_stream.cc b/net/websockets/websocket_http2_handshake_stream.cc
--- a/net/websockets/websocket_http2_handshake_stream.cc
+++ b/net/websockets/websocket_http2_handshake_stream.cc
@@ -33,6 +33,8 @@
 #include "net/websockets/websocket_handshake_constants.h"
 #include "net/websockets/websocket_handshake_request_info.h"
 
+#define ValidateStatus ValidateStatus_WebSocketHttp2HandshakeStream
+
 namespace net {
 
 namespace {
@@ -405,3 +407,5 @@
 }
 
 }  // namespace net
+
+#undef ValidateStatus
diff -uNr a/net/websockets/websocket_http3_handshake_stream.cc b/net/websockets/websocket_http3_handshake_stream.cc
--- a/net/websockets/websocket_http3_handshake_stream.cc
+++ b/net/websockets/websocket_http3_handshake_stream.cc
@@ -29,6 +29,8 @@
 #include "net/websockets/websocket_handshake_constants.h"
 #include "net/websockets/websocket_handshake_request_info.h"
 
+#define ValidateStatus ValidateStatus_WebSocketHttp3HandshakeStream
+
 namespace net {
 struct AlternativeService;
 
@@ -393,3 +395,5 @@
 }
 
 }  // namespace net
+
+#undef ValidateStatus
